testsuite/tests/kuadrant/authorino/metadata/test_user_info.py

```diff
@@ -34,6 +34,6 @@ def test_correct_auth(client, auth):
 
 def test_incorrect_auth(client, rhsso, user2):
     """Updates RHSSO user email address and tests incorrect auth"""
-    auth = HttpxOidcClientAuth(rhsso.get_token(user2.username, user2.password), "authorization")
+    auth = HttpxOidcClientAuth.from_user(rhsso.get_token, user2, "authorization")
```

Comment:
remove "authorization"

Reply:
It is a default so it can be omitted but keep it it should do no harm

Reply:
OK